components/screens/menu/Version.ts

```diff
@@ -0,0 +1,15 @@
+import * as Application from 'expo-application';
+import * as Updates from 'expo-updates';
+import {getUpdateTimeAsVersionString} from 'hooks/useEASUpdateStatus';
```

Comment:
You'll need to rebase - I changed this up recently, sorry!

Reply:
whoops, sorry I missed that